pom.xml

```diff
@@ -265,6 +265,9 @@
     <version.com.github.javaparser>3.13.10</version.com.github.javaparser>
 
     <version.org.apache.commons.csv>1.6</version.org.apache.commons.csv>
+
+    <!-- override needed for 7.4.0.CR2 -->
```

Comment:
please don't add such comments into upstream here ;)

Reply:
when the full downstream build is green. before merging this I will remove this comment